assisted-events-scrape/tests/integration/test_integration.py

```diff
@@ -0,0 +1,34 @@
+from config import ElasticsearchConfig
+import elasticsearch
+import waiting
+import time
+
+INIT_DELAY_SECONDS = 10
+ALL_EVENTS_NUMBER = 101
+
+
+class TestIntegration:
+    def setup(self):
+        self._wait_for_elastic()
+
+    def test_all_docs(self):
+        n_documents = self._es_client.count(index=self._config.index)['count']
+        assert(n_documents == ALL_EVENTS_NUMBER)
+
+    def _wait_for_elastic(self):
+        waiting.wait(
+            lambda: self._init_elastic_client(),
```

Comment:
this doesn't work? (didn't check, just wondered)
```suggestion
            self._init_elastic_client,
```